docs/docs/layout-animations/layout-animation-config.mdx

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_position: 5
+---
+
+# Skipping Layout Animations
+
+`LayoutAnimationConfig` is a wrapper that lets you skip entering and exiting animations.
+
+## Reference
+
+```javascript
+import Animated, {
+  LayoutAnimationConfig,
+  PinwheelIn,
+  PinwheelOut,
+} from 'react-native-reanimated';
+
+function App() {
+  const [show, setShow] = React.useState(true);
+
+  return (
+    <LayoutAnimationConfig skipEntering>
+      <View>
+        {show && <Animated.View entering={PinwheelIn} exiting={PinwheelOut} />}
+      </View>
+    </LayoutAnimationConfig>
```

Comment:
```suggestion
    // highlight-next-line
    <LayoutAnimationConfig skipEntering>
      <View>
        {show && <Animated.View entering={PinwheelIn} exiting={PinwheelOut} />}
      </View>
      // highlight-next-line
    </LayoutAnimationConfig>
```

Let's highlight the most important part

![814hrz](https://github.com/software-mansion/react-native-reanimated/assets/39658211/220e5bfe-3f50-43ab-a11d-2c7d2dda630f)